src/Core/AdminConsole/Services/Implementations/OrganizationService.cs

```diff
@@ -1430,7 +1360,7 @@ public async Task<List<Tuple<OrganizationUser, string>>> ConfirmUsersAsync_vNext
 
         var organization = await GetOrgById(organizationId);
         var allUsersOrgs = await _organizationUserRepository.GetManyByManyUsersAsync(validSelectedUserIds);
-        var users = await _userRepository.GetManyWithCalculatedPremiumAsync(validSelectedUserIds);
+        var users = await _userRepository.GetManyAsync(validSelectedUserIds);
```

Comment:
I'm guessing that `CalculatedPremium` is not required because that's already included in the new 2FA query?

Reply:
Excellent observation! `CalculatedPremium` isn't used here, I added it by mistake here https://github.com/bitwarden/server/pull/4524/files#diff-443d95443018d2d3daf6bd19935e90c1d57489f788a64e7d790bfb8f5a68b021R1566. You can see that by comparing to the original `CheckPolicies` method there.